checkbox/operations/index.js

```diff
@@ -0,0 +1,86 @@
+var Checkbox = require('./../index');
+
+/**
+ * Animates a circle ripple behind checkbox when pressed and held.
+ * @param  {Event} event that started ripple animation.
+ */
+Checkbox.prototype._triggerAnimation = function (event) {
+  this.model.set('visible', true);
+
+  this.frameCount_ = 1;
+  var bound = event.currentTarget.lastChild.getBoundingClientRect();
+  var x;
+  var y;
+
+  x = Math.round(event.clientX - bound.left);
+  y = Math.round(event.clientY - bound.top);
+
+  this._setRippleXY(x, y);
+  this._setRippleStyles(true);
+
+  window.requestAnimationFrame(this._animationFrameHandler.bind(this));
+};
+
+/**
+ * Remove ripple circle.
+ */
+Checkbox.prototype._removeAnimation = function () {
+  this.model.set('visible', false);
+}
+
+/**
+ * Handles an animation frame.
+ */
+Checkbox.prototype._animationFrameHandler = function () {
+  if (this.frameCount_-- > 0) {
+      window.requestAnimationFrame(this._animationFrameHandler.bind(this));
+  } else {
+      this._setRippleStyles(false);
+  }
+};
+
+/**
+ * Sets the ripple X and Y coordinates.
+ * @param  {number} newX the new X coordinate
+ * @param  {number} newY the new Y coordinate
+ */
+Checkbox.prototype._setRippleXY = function (newX, newY) {
+  this.x_ = newX;
```

Comment:
Why the naming convention of x_ and y_ ? Why not just x and y? I don't see any possible collisions?